2018-10-16 18:35:16,494 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3644860
2018-10-16 18:35:16,506 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-16 18:35:16,507 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-16 18:35:16,507 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-16 18:35:16,528 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-16 18:35:16,528 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-16 18:35:16,529 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-16 18:35:16,529 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-16 18:35:16,529 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-16 18:35:16,529 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-16 18:35:16,575 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-16 18:35:16,576 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-16 18:35:16,577 transport.py[line:1746][DEBUG]->Adding ssh-ed25519 host key for 47.94.35.234: b'b7e6b1ee10799b753c421f2887d6201a'
2018-10-16 18:35:16,583 transport.py[line:1746][DEBUG]->Trying discovered key b'6f9238ca1b3ee853c4a88bf9eb4ce8ae' in C:\Users\zbs/.ssh/id_rsa
2018-10-16 18:35:16,622 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:35:16,635 transport.py[line:1746][INFO]->Authentication (publickey) failed.
2018-10-16 18:35:16,669 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:35:16,684 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-16 18:35:16,697 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x2d93828
2018-10-16 18:35:16,697 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-16 18:35:16,706 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-16 18:35:16,707 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-16 18:35:16,719 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-16 18:35:16,719 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-16 18:35:16,720 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-16 18:35:16,720 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-16 18:35:16,720 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-16 18:35:16,720 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-16 18:35:16,730 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-16 18:35:16,731 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-16 18:35:16,732 transport.py[line:1746][DEBUG]->Attempting password auth...
2018-10-16 18:35:16,775 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:35:16,789 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-16 18:35:16,789 channel.py[line:1194][DEBUG]->[chan 0] Max packet in: 32768 bytes
2018-10-16 18:35:16,863 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-16 18:35:16,864 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-16 18:35:16,889 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-16 18:35:16,890 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-16 18:35:17,063 channel.py[line:1194][DEBUG]->[chan 0] Max packet out: 32768 bytes
2018-10-16 18:35:17,063 transport.py[line:1746][DEBUG]->Secsh channel 0 opened.
2018-10-16 18:35:17,068 channel.py[line:1194][DEBUG]->[chan 0] Sesch channel 0 request ok
2018-10-16 18:35:17,074 sftp.py[line:158][INFO]->[chan 0] Opened sftp connection (server version 3)
2018-10-16 18:35:27,176 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3d0b588
2018-10-16 18:35:27,176 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-16 18:35:27,185 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-16 18:35:27,185 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-16 18:35:27,193 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-16 18:35:27,193 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-16 18:35:27,194 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-16 18:35:27,194 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-16 18:35:27,194 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-16 18:35:27,194 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-16 18:35:27,203 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-16 18:35:27,203 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-16 18:35:27,204 transport.py[line:1746][DEBUG]->Adding ssh-ed25519 host key for 47.93.182.210: b'8c1df5aafd0341c959935d5c4b8e5806'
2018-10-16 18:35:27,205 transport.py[line:1746][DEBUG]->Trying discovered key b'6f9238ca1b3ee853c4a88bf9eb4ce8ae' in C:\Users\zbs/.ssh/id_rsa
2018-10-16 18:35:27,248 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:35:27,256 transport.py[line:1746][INFO]->Authentication (publickey) failed.
2018-10-16 18:35:27,261 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:35:27,272 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-16 18:35:27,276 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3d1d4a8
2018-10-16 18:35:27,277 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-16 18:35:27,284 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-16 18:35:27,284 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-16 18:35:27,290 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-16 18:35:27,290 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-16 18:35:27,290 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-16 18:35:27,290 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-16 18:35:27,291 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-16 18:35:27,291 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-16 18:35:27,300 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-16 18:35:27,300 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-16 18:35:27,301 transport.py[line:1746][DEBUG]->Attempting password auth...
2018-10-16 18:35:27,345 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:35:27,360 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-16 18:35:27,360 channel.py[line:1194][DEBUG]->[chan 0] Max packet in: 32768 bytes
2018-10-16 18:35:27,439 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-16 18:35:27,439 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-16 18:35:27,476 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-16 18:35:27,476 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-16 18:35:27,642 channel.py[line:1194][DEBUG]->[chan 0] Max packet out: 32768 bytes
2018-10-16 18:35:27,642 transport.py[line:1746][DEBUG]->Secsh channel 0 opened.
2018-10-16 18:35:27,647 channel.py[line:1194][DEBUG]->[chan 0] Sesch channel 0 request ok
2018-10-16 18:35:27,651 sftp.py[line:158][INFO]->[chan 0] Opened sftp connection (server version 3)
2018-10-16 18:35:27,652 transport.py[line:1746][DEBUG]->EOF in transport thread
2018-10-16 18:35:27,652 transport.py[line:1746][DEBUG]->EOF in transport thread
2018-10-16 18:35:27,653 transport.py[line:1746][DEBUG]->EOF in transport thread
2018-10-16 18:40:48,313 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x36448d0
2018-10-16 18:40:48,313 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-16 18:40:48,314 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-16 18:40:48,314 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-16 18:40:48,323 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-16 18:40:48,323 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-16 18:40:48,324 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-16 18:40:48,324 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-16 18:40:48,324 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-16 18:40:48,324 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-16 18:40:48,373 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-16 18:40:48,374 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-16 18:40:48,379 transport.py[line:1746][DEBUG]->Adding ssh-ed25519 host key for 47.94.35.234: b'b7e6b1ee10799b753c421f2887d6201a'
2018-10-16 18:40:48,386 transport.py[line:1746][DEBUG]->Trying discovered key b'6f9238ca1b3ee853c4a88bf9eb4ce8ae' in C:\Users\zbs/.ssh/id_rsa
2018-10-16 18:40:48,418 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:40:48,430 transport.py[line:1746][INFO]->Authentication (publickey) failed.
2018-10-16 18:40:48,436 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:40:48,449 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-16 18:40:48,472 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3cd37b8
2018-10-16 18:40:48,473 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-16 18:40:48,473 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-16 18:40:48,474 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-16 18:40:48,486 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-16 18:40:48,486 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-16 18:40:48,487 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-16 18:40:48,487 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-16 18:40:48,487 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-16 18:40:48,487 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-16 18:40:48,543 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-16 18:40:48,544 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-16 18:40:48,544 transport.py[line:1746][DEBUG]->Attempting password auth...
2018-10-16 18:40:48,590 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:40:48,604 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-16 18:40:48,604 channel.py[line:1194][DEBUG]->[chan 0] Max packet in: 32768 bytes
2018-10-16 18:40:48,654 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-16 18:40:48,654 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-16 18:40:48,682 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-16 18:40:48,683 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-16 18:40:48,884 channel.py[line:1194][DEBUG]->[chan 0] Max packet out: 32768 bytes
2018-10-16 18:40:48,885 transport.py[line:1746][DEBUG]->Secsh channel 0 opened.
2018-10-16 18:40:48,892 channel.py[line:1194][DEBUG]->[chan 0] Sesch channel 0 request ok
2018-10-16 18:40:48,898 sftp.py[line:158][INFO]->[chan 0] Opened sftp connection (server version 3)
2018-10-16 18:40:49,043 transport.py[line:1746][DEBUG]->EOF in transport thread
2018-10-16 18:42:40,327 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3643908
2018-10-16 18:42:40,328 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-16 18:42:40,328 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-16 18:42:40,329 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-16 18:42:40,338 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-16 18:42:40,338 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-16 18:42:40,338 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-16 18:42:40,338 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-16 18:42:40,339 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-16 18:42:40,339 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-16 18:42:40,390 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-16 18:42:40,391 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-16 18:42:40,399 transport.py[line:1746][DEBUG]->Adding ssh-ed25519 host key for 47.94.35.234: b'b7e6b1ee10799b753c421f2887d6201a'
2018-10-16 18:42:40,405 transport.py[line:1746][DEBUG]->Trying discovered key b'6f9238ca1b3ee853c4a88bf9eb4ce8ae' in C:\Users\zbs/.ssh/id_rsa
2018-10-16 18:42:40,701 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:42:40,711 transport.py[line:1746][INFO]->Authentication (publickey) failed.
2018-10-16 18:42:40,717 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:42:40,729 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-16 18:42:40,735 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3f16d30
2018-10-16 18:42:40,736 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-16 18:42:40,746 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-16 18:42:40,746 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-16 18:42:40,754 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-16 18:42:40,754 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-16 18:42:40,754 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-16 18:42:40,754 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-16 18:42:40,754 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-16 18:42:40,755 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-16 18:42:40,764 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-16 18:42:40,764 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-16 18:42:40,765 transport.py[line:1746][DEBUG]->Attempting password auth...
2018-10-16 18:42:40,810 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:42:40,823 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-16 18:42:40,824 channel.py[line:1194][DEBUG]->[chan 0] Max packet in: 32768 bytes
2018-10-16 18:42:40,889 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-16 18:42:40,890 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-16 18:42:40,925 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-16 18:42:40,925 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-16 18:42:41,094 channel.py[line:1194][DEBUG]->[chan 0] Max packet out: 32768 bytes
2018-10-16 18:42:41,095 transport.py[line:1746][DEBUG]->Secsh channel 0 opened.
2018-10-16 18:42:41,102 channel.py[line:1194][DEBUG]->[chan 0] Sesch channel 0 request ok
2018-10-16 18:42:41,109 sftp.py[line:158][INFO]->[chan 0] Opened sftp connection (server version 3)
2018-10-16 18:42:41,219 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3f2b9e8
2018-10-16 18:42:41,220 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-16 18:42:41,229 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-16 18:42:41,229 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-16 18:42:41,235 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-16 18:42:41,235 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-16 18:42:41,235 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-16 18:42:41,235 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-16 18:42:41,236 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-16 18:42:41,236 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-16 18:42:41,243 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-16 18:42:41,243 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-16 18:42:41,244 transport.py[line:1746][DEBUG]->Adding ssh-ed25519 host key for 47.93.182.210: b'8c1df5aafd0341c959935d5c4b8e5806'
2018-10-16 18:42:41,245 transport.py[line:1746][DEBUG]->Trying discovered key b'6f9238ca1b3ee853c4a88bf9eb4ce8ae' in C:\Users\zbs/.ssh/id_rsa
2018-10-16 18:42:41,292 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:42:41,301 transport.py[line:1746][INFO]->Authentication (publickey) failed.
2018-10-16 18:42:41,306 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:42:41,318 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-16 18:42:41,323 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3f3b5c0
2018-10-16 18:42:41,324 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-16 18:42:41,338 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-16 18:42:41,338 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-16 18:42:41,344 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-16 18:42:41,344 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-16 18:42:41,344 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-16 18:42:41,345 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-16 18:42:41,345 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-16 18:42:41,345 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-16 18:42:41,355 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-16 18:42:41,356 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-16 18:42:41,357 transport.py[line:1746][DEBUG]->Attempting password auth...
2018-10-16 18:42:41,406 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-16 18:42:41,422 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-16 18:42:41,423 channel.py[line:1194][DEBUG]->[chan 0] Max packet in: 32768 bytes
2018-10-16 18:42:41,500 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-16 18:42:41,501 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-16 18:42:41,516 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-16 18:42:41,516 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-16 18:42:41,703 channel.py[line:1194][DEBUG]->[chan 0] Max packet out: 32768 bytes
2018-10-16 18:42:41,703 transport.py[line:1746][DEBUG]->Secsh channel 0 opened.
2018-10-16 18:42:41,710 channel.py[line:1194][DEBUG]->[chan 0] Sesch channel 0 request ok
2018-10-16 18:42:41,718 sftp.py[line:158][INFO]->[chan 0] Opened sftp connection (server version 3)
2018-10-16 18:42:41,719 transport.py[line:1746][DEBUG]->EOF in transport thread
2018-10-16 18:42:41,719 transport.py[line:1746][DEBUG]->EOF in transport thread
2018-10-17 10:36:18,569 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3644908
2018-10-17 10:36:18,570 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-17 10:36:18,570 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-17 10:36:18,570 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-17 10:36:18,614 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-17 10:36:18,615 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-17 10:36:18,615 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-17 10:36:18,615 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-17 10:36:18,615 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-17 10:36:18,615 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-17 10:36:18,684 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-17 10:36:18,684 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-17 10:36:18,685 transport.py[line:1746][DEBUG]->Adding ssh-ed25519 host key for 47.94.35.234: b'b7e6b1ee10799b753c421f2887d6201a'
2018-10-17 10:36:18,694 transport.py[line:1746][DEBUG]->Trying discovered key b'6f9238ca1b3ee853c4a88bf9eb4ce8ae' in C:\Users\zbs/.ssh/id_rsa
2018-10-17 10:36:18,732 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-17 10:36:18,745 transport.py[line:1746][INFO]->Authentication (publickey) failed.
2018-10-17 10:36:18,763 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-17 10:36:18,775 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-17 10:36:18,793 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3ee6d30
2018-10-17 10:36:18,794 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-17 10:36:18,794 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-17 10:36:18,794 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-17 10:36:18,917 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-17 10:36:18,917 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-17 10:36:18,918 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-17 10:36:18,918 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-17 10:36:18,918 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-17 10:36:18,918 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-17 10:36:18,933 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-17 10:36:18,934 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-17 10:36:18,935 transport.py[line:1746][DEBUG]->Attempting password auth...
2018-10-17 10:36:19,000 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-17 10:36:19,000 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-17 10:36:19,051 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-17 10:36:19,066 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-17 10:36:19,067 channel.py[line:1194][DEBUG]->[chan 0] Max packet in: 32768 bytes
2018-10-17 10:36:19,133 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-17 10:36:19,133 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-17 10:36:19,345 channel.py[line:1194][DEBUG]->[chan 0] Max packet out: 32768 bytes
2018-10-17 10:36:19,345 transport.py[line:1746][DEBUG]->Secsh channel 0 opened.
2018-10-17 10:36:19,349 channel.py[line:1194][DEBUG]->[chan 0] Sesch channel 0 request ok
2018-10-17 10:36:19,352 sftp.py[line:158][INFO]->[chan 0] Opened sftp connection (server version 3)
2018-10-17 10:36:29,465 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3efb5f8
2018-10-17 10:36:29,465 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-17 10:36:29,473 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-17 10:36:29,473 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-17 10:36:29,479 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-17 10:36:29,479 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-17 10:36:29,479 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-17 10:36:29,479 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-17 10:36:29,479 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-17 10:36:29,479 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-17 10:36:29,488 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-17 10:36:29,489 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-17 10:36:29,489 transport.py[line:1746][DEBUG]->Adding ssh-ed25519 host key for 47.93.182.210: b'8c1df5aafd0341c959935d5c4b8e5806'
2018-10-17 10:36:29,490 transport.py[line:1746][DEBUG]->Trying discovered key b'6f9238ca1b3ee853c4a88bf9eb4ce8ae' in C:\Users\zbs/.ssh/id_rsa
2018-10-17 10:36:29,549 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-17 10:36:29,571 transport.py[line:1746][INFO]->Authentication (publickey) failed.
2018-10-17 10:36:29,583 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-17 10:36:29,603 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-17 10:36:29,616 transport.py[line:1746][DEBUG]->starting thread (client mode): 0x3f0c4e0
2018-10-17 10:36:29,617 transport.py[line:1746][DEBUG]->Local version/idstring: SSH-2.0-paramiko_2.4.2
2018-10-17 10:36:29,635 transport.py[line:1746][DEBUG]->Remote version/idstring: SSH-2.0-OpenSSH_7.2p2 Ubuntu-4ubuntu2.4
2018-10-17 10:36:29,635 transport.py[line:1746][INFO]->Connected (version 2.0, client OpenSSH_7.2p2)
2018-10-17 10:36:29,650 transport.py[line:1746][DEBUG]->kex algos:['curve25519-sha256@libssh.org', 'ecdh-sha2-nistp256', 'ecdh-sha2-nistp384', 'ecdh-sha2-nistp521', 'diffie-hellman-group-exchange-sha256', 'diffie-hellman-group14-sha1'] server key:['ssh-rsa', 'rsa-sha2-512', 'rsa-sha2-256', 'ecdsa-sha2-nistp256', 'ssh-ed25519'] client encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] server encrypt:['chacha20-poly1305@openssh.com', 'aes128-ctr', 'aes192-ctr', 'aes256-ctr', 'aes128-gcm@openssh.com', 'aes256-gcm@openssh.com'] client mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] server mac:['umac-64-etm@openssh.com', 'umac-128-etm@openssh.com', 'hmac-sha2-256-etm@openssh.com', 'hmac-sha2-512-etm@openssh.com', 'hmac-sha1-etm@openssh.com', 'umac-64@openssh.com', 'umac-128@openssh.com', 'hmac-sha2-256', 'hmac-sha2-512', 'hmac-sha1'] client compress:['none', 'zlib@openssh.com'] server compress:['none', 'zlib@openssh.com'] client lang:[''] server lang:[''] kex follows?False
2018-10-17 10:36:29,650 transport.py[line:1746][DEBUG]->Kex agreed: ecdh-sha2-nistp256
2018-10-17 10:36:29,650 transport.py[line:1746][DEBUG]->HostKey agreed: ssh-ed25519
2018-10-17 10:36:29,650 transport.py[line:1746][DEBUG]->Cipher agreed: aes128-ctr
2018-10-17 10:36:29,651 transport.py[line:1746][DEBUG]->MAC agreed: hmac-sha2-256
2018-10-17 10:36:29,651 transport.py[line:1746][DEBUG]->Compression agreed: none
2018-10-17 10:36:29,671 transport.py[line:1746][DEBUG]->kex engine KexNistp256 specified hash_algo <built-in function openssl_sha256>
2018-10-17 10:36:29,671 transport.py[line:1746][DEBUG]->Switch to new keys ...
2018-10-17 10:36:29,672 transport.py[line:1746][DEBUG]->Attempting password auth...
2018-10-17 10:36:29,721 transport.py[line:1746][DEBUG]->userauth is OK
2018-10-17 10:36:29,736 transport.py[line:1746][INFO]->Authentication (password) successful!
2018-10-17 10:36:29,736 channel.py[line:1194][DEBUG]->[chan 0] Max packet in: 32768 bytes
2018-10-17 10:36:29,809 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-17 10:36:29,809 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-17 10:36:29,812 transport.py[line:1746][DEBUG]->Received global request "hostkeys-00@openssh.com"
2018-10-17 10:36:29,812 transport.py[line:1746][DEBUG]->Rejecting "hostkeys-00@openssh.com" global request from server.
2018-10-17 10:36:30,018 channel.py[line:1194][DEBUG]->[chan 0] Max packet out: 32768 bytes
2018-10-17 10:36:30,018 transport.py[line:1746][DEBUG]->Secsh channel 0 opened.
2018-10-17 10:36:30,024 channel.py[line:1194][DEBUG]->[chan 0] Sesch channel 0 request ok
2018-10-17 10:36:30,031 sftp.py[line:158][INFO]->[chan 0] Opened sftp connection (server version 3)
2018-10-17 10:36:30,032 transport.py[line:1746][DEBUG]->EOF in transport thread
2018-10-17 10:36:30,033 transport.py[line:1746][DEBUG]->EOF in transport thread
